wdio-ui5-service/src/lib/WDI5.js

```diff
@@ -579,6 +579,30 @@ module.exports = class WDI5 {
             // further processing there
             controlSelector.selector.id = controlSelector.selector.id.toString();
         }
+
+        // check whether we have a (partial) text matcher
+        // that should match:
+        // properties: {
+        //     text: new RegExp(/.*ersi.*/gm)
+        // }
+        // ...but not:
```

Comment:
Suggestion: Add comment or or README for declarative matcher in UI5 version < 1.74.